demos/twa-webview-fallback/src/main/java/com/google/browser/examples/twawebviewfallback/WebViewFallbackActivity.java

```diff
@@ -30,15 +31,30 @@
 public class WebViewFallbackActivity extends AppCompatActivity {
     public static final String KEY_LAUNCH_URI =
             "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity.KEY_LAUNCH_URL";
+    public static final String NAVIGATION_BAR_COLOR =
+            "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity" +
+                    ".NAVIGATION_BAR_COLOR";
+    public static final String STATUS_BAR_COLOR =
+            "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity" +
+                    ".STATUS_BAR_COLOR";
+
 
     private Uri mLaunchUrl;
+    private int mStatusBarColor;
 
+    @SuppressLint("SetJavaScriptEnabled")
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         this.mLaunchUrl = this.getIntent().getParcelableExtra(KEY_LAUNCH_URI);
 
+        int navigationBarColor = this.getIntent().getIntExtra(NAVIGATION_BAR_COLOR, 0);
+        getWindow().setNavigationBarColor(navigationBarColor);
```

Comment:
Does this cause a behaviour change when an app doesn't have the colours set?

Reply:
Changed to only set colors if they exist in the Intent

Reply:
But that still would cause a behaviour change because there's always going to be the extras in the Intent. (We put the defaults from LauncherActivityMetadata in there.)

This isn't necessarily a bad thing though - maybe we want to do it.

Reply:
So, it seems we'd be replicating the behaviour that have today? I think this makes sense. The alternative would be updating `LauncherActivityMetadata` to get its default values from the system. If we did, the demo would automatically get the new value.

Reply:
Fair enough, that's fine by me.